homeassistant/components/onkyo/media_player.py

```diff
@@ -385,7 +390,7 @@ def update(self):
         volume_raw = self.command("zone{}.volume=query".format(self._zone))
         mute_raw = self.command("zone{}.muting=query".format(self._zone))
         current_source_raw = self.command("zone{}.selector=query".format(self._zone))
-
+        preset_raw = self.command("zone{}.preset=query".format(self._zone))
```

Comment:
Please use f-strings instead of str.format.
```suggestion
        preset_raw = self.command(f"zone{self._zone}.preset=query")
```